spec/models/user_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe User do
+  it_behaves_like "has a uuid"
```

Comment:
Interesting. I will leave it in and add a comment explaining why we're diverging from the norm in this case.